themes/ht/ht.theme.sh

```diff
@@ -0,0 +1,80 @@
+#! bash oh-my-bash.module
+
+# Harrison's Theme (ht)
+
+# Description:
+# Simple prompt that shows basic working directory and exit code info
+# Works with 8 color terminals
+
+# Example prompt:
+# ● [harrison@ubN2] in src ± |dev ✗|
+
+function exit_color {
+  case $1 in
+    0)
+      echo $_omb_prompt_green
+      ;;
+    1)
+      echo $_omb_prompt_red
+      ;;
+    2)
+      echo $_omb_prompt_gray
+      ;;
+    126)
+      echo $_omb_prompt_cyan
+      ;;
+    127)
+      echo $_omb_prompt_magenta
+      ;;
+    130)
+      echo $_omb_prompt_black
+      ;;
+    148)
+      echo $_omb_prompt_yellow
+      ;;
+    *)
+      echo $_omb_prompt_blue
+      ;;
+  esac
+}
+
+# Displays the current prompt
+function _omb_theme_PROMPT_COMMAND() {
+  # Capture exit code
+  # NOTE: DO NOT MOVE
+  local EXIT_CODE=$?
+
+  # Start prompt blank
+  PS1=""
+
+  # Exit code indicator
+  PS1+="$(exit_color "$EXIT_CODE")●${_omb_prompt_reset_color}"
+
+  # Environment info
+  PS1+=" ${_omb_prompt_red}$(_omb_prompt_get_ruby_env)${_omb_prompt_reset_color}"
+  PS1+=" ${_omb_prompt_green}$(_omb_prompt_get_python_env)${_omb_prompt_reset_color}"
+
+  # User and host
+  local user_host_prefix=" ${_omb_prompt_reset_color}["
+  local user_host_suffix="${_omb_prompt_reset_color}]"
+  local user="${_omb_prompt_blue}\u"
+  local host="${_omb_prompt_cyan}\H"
+  local at="${_omb_prompt_reset_color}@"
+  PS1+="${user_host_prefix}${user}${at}${host}${user_host_suffix}"
+
+  # Working directory
+  PS1+=" in ${_omb_prompt_magenta}\W${_omb_prompt_reset_color}"
+
+  # SCM
+  local SCM_GIT_SHOW_MINIMAL_INFO="true"
+  local SCM_THEME_PROMPT_DIRTY=" ${_omb_prompt_red}✗${_omb_prompt_normal}"
+  local SCM_THEME_PROMPT_CLEAN=" ${_omb_prompt_green}✓${_omb_prompt_normal}"
+  local SCM_NONE_CHAR=""
```

Comment:
Can we set these variables globally so that the users of `ht` can override them?